src/transformState.js

```diff
@@ -1,11 +1,51 @@
 'use strict';
-
 /**
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  const addProperties = 'addProperties';
+  const clear = 'clear';
+  const removeProperties = 'removeProperties';
```

Comment:
better to use object here
const `actions = { addProperties: 'addProperties', ....}`